src/configuration/structures.ts

```diff
@@ -238,3 +238,11 @@ export interface Tag {
   author_id: string;
   created_at: string;
 }
+
+export interface GroupProperties {
+  id: string;
+  properties: Properties
+}
+export interface GroupsProperties {
+  list_groups_properties: GroupProperties[]
+}
```

Comment:
```suggestion
export type GroupsProperties = []GroupProperties
```

or you could just return `Promise<[]GroupProperties>` in `listGroupsProperties` method.